docs/data-tunnel/route-types/01-tss.md

```diff
@@ -1,13 +1,91 @@
 
-> This route type is currently only available on BandChain v3 testnet.
+A target contract must implement the `IDataConsumer` interface to support the `BandTunnelRouter` contract. [This contract](https://holesky.etherscan.io/address/0x4eabfc3cd1d9b1170707debd9fe5ab8f657f7a3f#code) is an example of the `DataConsumer` contract that support `IDataConsumer` interface. The implementation details are provided below:
+
+```solidity
+interface IDataConsumer {
+    /**
+     * @dev Processes the relayed message.
+     *
+     * The relayed message must be evaluated from the tunnelRouter contract and
+     * verified by the tssVerifier contract before forwarding to the target contract.
+     *
+     * @param data The decoded tss message that is relayed from the tunnelRouter contract.
+     */
+    function process(PacketDecoder.TssMessage memory data) external;
+
+    /**
+     * @dev Activates the tunnel and set the sequence on tunnelRouter contract.
+     *
+     * This function deposits tokens into the vault and sets the latest sequence on the
+     * tunnelRouter contract if the current deposit in the vault contract exceeds a threshold.
+     * The transaction is reverted if the threshold is not met.
+     *
+     * This function should be called by the contract owner.
+     *
+     * @param latestSeq The new sequence of the tunnel.
+     */
+    function activate(uint64 latestSeq) external payable;
+
+    /**
+     * @dev Returns The tunnelRouter contract address.
+     */
+    function tunnelRouter() external view returns (address);
+
+    /**
+     * @dev Returns The tunnelId of the contract address.
+     */
+    function tunnelId() external view returns (uint64);
+}
+```
+
+### Creating a TSS Tunnel in BandChain
+
+To create a TSS Tunnel using the command-line interface (CLI), execute the following command:
+
+```bash
+bandd tx tunnel create-tunnel tss [destination-chain-id] [destination-contract-address] [encoder] [initial-deposit] [interval] [signal-deviations-json-file]
+```
+
+Parameters:
+
+- **destination-chain-id**: The ID of the destination chain supported by Band Protocol. A list of supported chains will be publicly provided soon.
+- **destination-contract-address**: The contract address on the destination chain that implements the `IDataConsumer` interface to support the `BandTunnelRouter` contract.
+- **encoder**: The type of data and the method of encoding for that data.
+- **initial-deposit**: The initial deposit required to create the tunnel.
+- **interval**: The frequency at which price updates are sent.
+- **signal-deviations-json-file**: A JSON file containing signal information for the tunnel.
+
+This command initializes a TSS Tunnel, enabling the secure transmission of real-time price data from **BandChain** to a target contract on another blockchain.
+
+Currently, there are 2 types of encoder:
+- FIXED_POINT_ABI (1) for requiring abi-encoded decimal price data, in the base of billion (x * 10^9), from the tunnel.
```

Comment:
Please confirm, I thought Oracle is (x * 10^18)

Reply:
it's 10^9